examples/step-1/step-1.cc

```diff
@@ -99,31 +100,40 @@ void second_grid ()
   GridGenerator::hyper_shell (triangulation,
                               center, inner_radius, outer_radius,
                               10);
-  // By default, the triangulation assumes that all boundaries are straight
-  // and given by the cells of the coarse grid (which we just created). It
-  // uses this information when cells at the boundary are refined and new
-  // points need to be introduced on the boundary; if the boundary is assumed
-  // to be straight, then new points will simply be in the middle of the
+  // By default, the triangulation assumes that all boundaries are
+  // straigth lines, and all cells are bi-linear quads, and that they
+  // are defined by the cells of the coarse grid (which we just
+  // created). It uses this information when cells are refined and new
+  // points need to be introduced; if the domain is assumed to be
+  // flat, then new points will simply be in the middle of the
```

Comment:
I think the word "flat" is maybe a bit out of place here because the reader is not thinking of this here in the context of manifolds etc. How about rewording this as follows:
```
// By default, the triangulation assumes that all boundaries are
+ // straight lines, and all cells are bi-linear quads, and that they
+ // are defined by the cells of the coarse grid (which we just
+ // created). Unless we do something special, when new
+ // points need to be introduced; the domain is assumed to be
+ // delineated by the straight lines of the coarse mesh, and new points will simply be in the middle of the
```